source/fundamentals/crud/write-operations/change-a-document.txt

```diff
@@ -0,0 +1,202 @@
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 1
+   :class: singlecol
+
+Overview
+--------
+
+You can change documents in a MongoDB collection using **update** and
+**replace** operations. Update operations modify only the fields that
+you specify in one or more documents while leaving other fields and
+values unchanged. Replace operations remove all existing fields in one
+or more documents and substitute the deleted fields with new fields and
+values.
+
+.. _updateDocuments:
+
+Update
+------
+
+To update one or more documents, create an **update document** that
+includes an **update operator**, which specifies the type of update to
+perform and the fields and values that describe the change. Update
+documents use the following format:
+
+.. code-block:: go
+
+   bson.D{{"<update operator>", bson.D{{"<field>", "<value>"}, {"<field>", "<value>"}, ... }}, {"<update operator>", ... }, ... }
+   
+
+You can use one or more of the following update operators in your update
+document:
+
+.. list-table::
+   :header-rows: 1
+   :widths: 40 60
+
+   * - Update Operator
+     - Description
+
+   * - ``$set``
+     - replaces the value of a field with a specified value or creates a new
+       field with a specified value
+
+   * - ``$inc``
+     - increments or decrements field values
+
+   * - ``$rename``
+     - renames fields
+
+   * - ``$unset``
+     - removes fields
+     
+   * - ``$mul``
+     - multiplies the value of a field by a specified number
+
+See the MongoDB server manual for a :manual:`complete list of update operators
+and descriptions </reference/operator/update-field/>`.
+
+.. note:: Aggregation Pipelines in Update Operations
+
+   If you are using MongoDB Version 4.2 or later, you can use aggregation
+   pipelines made up of a subset of aggregation stages in update operations. For
+   more information on the aggregation stages MongoDB supports in
+   aggregation pipelines, see our tutorial on performing
+   :manual:`updates with aggregation pipelines </tutorial/update-documents-with-aggregation-pipeline/>`.
+
+Example
+~~~~~~~
+
+The following document describes an employee:
+
+.. code-block:: json
+   :copyable: false
+
+   {
+      "_id" : 2158,
+      "name" : "Mary Shelley",
+      "department" : "Marketing",
+      "role" : "Marketing Analyst",
+      "bonus" : 2500,
+      ...
+   }
+
+The following code shows how you can use the ``$set`` update operator to
+change the value of the ``role`` field to "Marketing Director" and the
+``$inc`` update operator to increment the value of the ``bonus`` field
+by 2000:
+
+.. code-block:: go
+
+   filter := bson.D{{"_id", 2158}}
+   update := bson.D{{"$set", bson.D{{"role", "Marketing Director"}}}, {"$inc", bson.D{{"bonus", 2000}}}}
+
+   result, err := collection.UpdateOne(context.TODO(), filter, update)
+
+The updated document should resemble the following, with updated values in
+the ``role`` and ``bonus`` fields and all other values unchanged:
```

Comment:
```suggestion
The following shows the updated document:
```
Suggestion: I don't think it's necessary to state the fields updated since it was just mentioned above.